geometry_msgs/Point centriod
shape_msgs/SolidPrimitive solid_shape

